### PR TITLE
Art.io update

### PR DESCRIPTION
**Pull request by amashinchi-rc**
**Details:**
Significant update to the atomicredteam.io site. Removed information that was stagnate or redundant. Focuses on directing viewers to the project they are looking for with short concise summaries. Removed dependancies on external images. Also updated/fixed the "roll-the-dice" widget. 

**Testing:**
Testing done locally with a local jekyll (Docker) instance.

Screen Shot 2021-07-30 at 10 42 04 AM

**Associated Issues:**
N/A

### CODE REVIEW

File: docs/atomicredteam.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+use to simulate adversarial activity in their environments.
+
+## Fast
+
+Atomic tests run in five minutes or less and require minimal setup. Spend less
+time configuring and more time testing!
+
+## Focused
+
+Security teams don't want to operate with a "hopes and prayers" attidude towards
```
Comment (mgraeber-rc) [inline docs/atomicredteam.md:17]: s/attidude/attitude/

File: docs/atomicredteam.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+detection. Atomic tests are mapped to the MITRE ATT&CK matrix, so you always
```
Comment (mgraeber-rc) [inline docs/atomicredteam.md:18]: I'd consider supplying a link to MITRE ATT&CK

File: docs/atomictestharnesses.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+---
+
+# AtomicTestHarnesses
+
+AtomicTestHarnesses is a PowerShell module that simulates multiple variations
+on a single attack technique.
```
Comment (mgraeber-rc) [inline docs/atomictestharnesses.md:8]: s/on a single/of a single/

File: docs/atomictestharnesses.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+## Complexity simplified
```
Comment (mgraeber-rc) [inline docs/atomictestharnesses.md:10]: s/simplified/reduced/

File: docs/atomictestharnesses.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+The exact details of a technique can vary from attack to attack.
+With AtomicTestHarnesses, you can test for multiple versions of an attack at
```
Comment (mgraeber-rc) [inline docs/atomictestharnesses.md:13]: s/versions/variations/

File: docs/index.md [hunk 6]
```diff
@@ -2,48 +2,39 @@
...
+over a network.
 
-With these principles in mind, we publicly launched Atomic Red Team. The response we received was, honestly, a bit
-overwhelming and showed us that there was a massive need in the community for this type of project. We are
-especially grateful to the MITRE ATT&CK team, whose great work has given us a great taxonomy to work within.
+## AtomicTestHarnesses
 
-# Key Beliefs
+A PowerShell module for executing many variations of an attack technique at
+once. AtomicTestHarnesses also include tests to validate test execution and
```
Comment (mgraeber-rc) [inline docs/index.md:25]: s/include/includes/

**Comment by amashinchi-rc** [comment]
@mattifestation thanks for the content review! Suggestions have been applied.

**Review by mgraeber-rc — APPROVED** [review]
Looks good! Thanks!

**Review by clr2of8 — APPROVED** [review]

**Pull request merged**